filesystem/azure.go

```diff
@@ -0,0 +1,196 @@
+package filesystem
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"github.com/MSOpenTech/azure-sdk-for-go/storage"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+)
+
+type AzureClient struct {
```

Comment:
AzureClient -> Client. 
So it will look like azure.Client instead of azure.AzureClient.